standard/statements.md

```diff
@@ -360,18 +360,15 @@ expression_statement
     ;
 
 statement_expression
```

Comment:
@BillWagner - Same as for member_declarator, here the latest change appears to replace statement_expression with only null-conditional options rather than add them to it. Has something gone awry or am I misreading this?